src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -45,10 +45,12 @@ class Wedge3D {
    * \param sphericity_of_other_surface Value between 0 and 1 which determines
    * whether the other surface is flat (value of 0), spherical (value of 1) or
    * somewhere in between
+   * \param with_equiangular_map Determines whether to apply a tangent function
+   * mapping to the logical coordinates (for 'true') or not (for 'false').
```

Comment:
@fmahebert 
@kidder 

Are there cases in which we would want to use an equidistant mapping for a spherical surface? If not, and equiangular mappings are always suited for spherical surfaces, should this boolean then be set automatically, depending on the value given for `sphericity`?

Note: As Wedge3D is currently written, it uses equidistant variables for the flat face, and then uses either equidistant or equiangular variables (depending on the bool) for the spherical face. 

I plan on adding the option to also use equiangular variables for the flat face, but should this choice be tied to choice of variables for the spherical face, or independent of it?

The way I was thinking about it, if they are independent, then in the case of a flat lower face and spherical upper face, I could use the equidistant variables on the flat face and the equiangular variables on the spherical face. Then we could accomplish having natural grids on both surfaces. The drawback is I don't know of any analytic inverse to such a map... 

There are advantages to uniformly applying/not applying the tangent map to the logical coordinates on both tangential surfaces (analytic inverse, radial grid lines). Is it known if this outweighs the advantages of having natural grids on both surfaces?

Reply:
Some answers, some rambling thoughts added in:

1. Are there cases in which we would want to use an equidistant mapping for a spherical surface?
I do not think we ever *prefer* an equidistant map on a spherical surface, because it can give larger error vs. an equiangular mapping. However, if the mapping choice is constrainted be uniform across the element, we may be forced to use the sub-optimal (for the sphere) equidistant map. My SpEC-DG TOV stars used a globally equidistant map for this reason — I had to pick whether to use a map better suited to the spherical outer regions or the cubical inner regions of my grid.

2. If not, and equiangular mappings are always suited for spherical surfaces, should this boolean then be set automatically, depending on the value given for sphericity?
I would say this depends on whether the map is uniformly applied/not applied:
* If the map is applied independently at each surface, then a spherical surface should automatically use an equiangular map. A choice must be made for the curvature value where the map should automatically switch, and because of this it may be nice to have manual override. Or perhaps we can smoothly interpolate between equiangular and equidistant mappings as the surface curvature varies...
* If the map is uniformly applied, then the grid will not conform to the natural equidistant coordinates on the central cube of a cube-to-sphere wedge. This means the central cube (i.e. `Brick`) should have an option for an equiangular mapping as well, or we have to deal with projecting the boundary conditions between the elements. Due to these tradeoffs, a uniformly applied map should probably not be turned on automatically, because it is not clear when the tradeoffs are acceptable.

3. I plan on adding the option to also use equiangular variables for the flat face, but should this choice be tied to choice of variables for the spherical face, or independent of it?
I cannot think of any reason to use an equiangular mapping on a flat face, besides attempting to match with an equiangular spherical surface (which should only come up if the mapping is uniformly applied).

4. Is it known if [the advantages of: analytic inverse, radial grid lines] outweighs the advantages of having natural grids on both surfaces?
I do not know for sure ¯\\\_(ツ)_/¯
My gut feeling (biased by dealing with the downsides of uniformly applied mappings (as in rants above)) is that using the natural mapping on each surface is the way to go, so that we don't have to debate the tradeoffs of equiangular vs. equidistant for every complex domain we will make in the future. I admit I do not have a good sense of how inconvenient it would be to lose the analytic inverse in SpECTRE...

Reply:
Thanks for your input :smiley: I too have been wondering about interpolating between equidistant and equiangular mappings depending on the value given for `sphericity`. I think having natural grids on both surfaces is worth exploring once we have numerical inverses in SpECTRE. After discussing with Larry some our first move will be to use the same variables on both surfaces (as well as the inner cube) and have an analytic inverse.

Reply:
I think this is a reasonable first step; it's no worse than SpEC 😛. It may be sensible to keep the analytic mapping simple anyway, and save the variation of the angular mapping with sphericity until transfinite interpolation maps.